Cargo.toml

```diff
@@ -25,4 +25,5 @@ default = ["termcolor"]
 [dependencies]
 log = { version = "0.4.*", features = ["std"] }
 termcolor = { version = "1.1.*", optional = true }
+paris = { version = "*", optional = true }
```

Comment:
Please don't depend with wildcard on paris. Crates with wildcard dependencies are denied by crates.io, please just use the latest version.